client/boot/index.js

```diff
@@ -1,5 +1,9 @@
 // Initialize localStorage polyfill before any dependencies are loaded
 require( 'lib/local-storage' )();
+
+// Initialize draft-js polyfill needed for SEO title format editor
+require( 'lib/draft-js-polyfill' )();
```

Comment:
We shouldn't be loading a very specific polyfill like this that affects all of calypso.